vendor.conf

```diff
@@ -25,7 +25,7 @@ google.golang.org/grpc v1.0.5
 github.com/pkg/errors v0.8.0
 github.com/nightlyone/lockfile 1d49c987357a327b5b03aa84cbddd582c328615d
 github.com/opencontainers/go-digest 21dfd564fd89c944783d00d069f33e3e7123c448
-golang.org/x/sys/unix d75a52659825e75fff6158388dddc6a5b04f9ba5
+golang.org/x/sys/unix f3918c30c5c2cb527c0b071a27c35120a6c0719a
```

Comment:
Doesn't need to change in this PR but note that this should be changed to `golang.org/x/sys`